lib/appoptics_apm/support/trace_parent.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) SolarWinds, LLC.
+# All rights reserved.
+
+module AppOpticsAPM
+  module TraceParent
+    # Regexp copied from Ruby OT trace_parent.rb
+    REGEXP = /^(?<version>[a-f0-9]{2})-(?<trace_id>[a-f0-9]{32})-(?<span_id>[a-f0-9]{16})-(?<flags>[a-f0-9]{2})$/.freeze
```

Comment:
The regex seems to use capture group naming, could these named captures be used to optimize and clarify the code in any of the methods below?

Reply:
they main advantage is that they make the regex more readable 
The regex is used in 2 places, valid? and extract_id (here the capture group is used)

The last 2 methods process an XTrace formatted id and will change